### PR TITLE
Ondivi integration doc

### PR DESCRIPTION
**Pull request by blablatdinov**
# I have made things!

Insert a recommendation in the documentation to use [ondivi](https://github.com/blablatdinov/ondivi) for legacy codebases

## Checklist

- [x] I have double checked that there are no unrelated changes in this pull request (old patches, accidental config files, etc)
- [ ] I have created at least one test case for the changes I have made
- [x] I have updated the documentation for the changes I have made
- [x] I have added my changes to the `CHANGELOG.md`

## Related issues

#2974

🙏 Please, if you or your company is finding wemake-python-styleguide valuable, help us sustain the project by sponsoring it transparently on https://opencollective.com/wemake-python-styleguide. As a thank you, your profile/company logo will be added to our main README which receives hundreds of unique visitors per day.

### CODE REVIEW

**Comment by blablatdinov** [comment]
@sobolevn take a look please. Feel free to any changes in text

I have double check that the configuration files are not corrupted as in #2520 😅

File: docs/pages/usage/integrations/index.rst [hunk 1]
```diff
@@ -6,6 +6,7 @@
   auto-formatters.rst
   nitpick.rst
   flakeheaven.rst
```
Comment (sobolevn) [inline docs/pages/usage/integrations/index.rst:8]: please, remove this file

Comment (blablatdinov) [reply docs/pages/usage/integrations/index.rst:8]: `flakeheaven.rst` file too?

Comment (sobolevn) [reply docs/pages/usage/integrations/index.rst:8]: I mean `flakeheaven.rst` :)

Comment (blablatdinov) [reply docs/pages/usage/integrations/index.rst]: Removed in [commit](https://github.com/wemake-services/wemake-python-styleguide/pull/2975/commits/f213364e6cc34a9d0335581817e840b72b41bfa1)

**Review by sobolevn — APPROVED** [review]
Thanks!

**Pull request merged**